builder/builder-next/builder.go

```diff
@@ -130,6 +145,9 @@ func (b *Builder) Prune(ctx context.Context, opts types.BuildCachePruneOptions)
 
 	validFilters := make(map[string]bool, 1+len(cacheFields))
 	validFilters["unused-for"] = true
+	validFilters["until"] = true
+	validFilters["label"] = true  // TODO(tiborvass): handle label
```

Comment:
Not sure why those two (`label` and `!label`) are here if they are not implemented yet :thinking:

Reply:
We could possibly do it on the cli side for now (https://github.com/docker/cli/pull/152)

Reply:
@vdemeester I should have commented in the code, but the idea is that it prevents from erroring out on `system prune --filter label=foobar`.

Reply:
I think having both would actually be best for now (so that a 18.09.0 cli connecting to a 18.09.1 daemon would work without getting errors)

So, I guess this is ok for now